config/database/migrations/20240903150136_accounts_table.js

```diff
@@ -7,7 +7,7 @@ export async function up(knex) {
     table.uuid('id').defaultTo(knex.fn.uuid()).primary();
     table.uuid('user_id').references('id').inTable('users');
     table.decimal('balance', 15, 2).notNullable().defaultTo(0.0);
-    table.boolean('is_default').defaultTo(true);
+    table.boolean('is_default').defaultTo(false);
```

Comment:
Updating a migration this way doesnt update your database o, you are supposed to create a new migration to update the column

Reply:
Please could you point me to how I can create a new migration to update a column. Knex is still a bit new to me.

What I usually do is rollback on the migration and then update the migration to run the batch again... Which clears everything then I have to start over